run/GCHP/setCommonRunSettings.sh.template

```diff
@@ -211,6 +211,21 @@ RRTMG_Timestep_sec=10800
 # Mass tuning factor is used in the HEMCO DustDead extenstion for GEOS-Chem
 # benchmarking and is resolution-dependent. We recommend using offline dust
```

Comment:
"and is resolution and met-source dependent."

Reply:
@lizziel: We have simplified the logic where mass tuning factors are set in commit d08e541762be53774d818deaf691a59df4890c47.